### ISSUE TITLE
Document approach required to view long strings

### ISSUE THREAD

**Issue by FairlySadPanda** [golang/vscode-go#2072]
**Is your feature request related to a problem? Please describe.**
Several tickets exist that query how to display lengthy strings inside the debugger frame.

These tickets are all locked, more recently with a comment similar to "dlv-dap doesn't support this yet".

**Describe the solution you'd like**
Document the approach a user is supposed to take to view truncated long strings.

**Describe alternatives you've considered**
Add this feature to dlv-dap. This seems like a lot of work for something that can be covered in the short term with documentation.

**Additional context**
Yes, there is probably an article or note somewhere that explains this, but if it doesn't appear in the top ten when searching Google it essentially does not exist to the average programmer.

**Comment by hyangah** [comment]
How long is the string you want to browse? 
https://github.com/golang/vscode-go/blob/master/docs/debugging.md#i-need-to-view-large-strings-how-can-i-do-that-if-dlvloadconfig-with-maxstringlen-is-deprecated

**Comment by FairlySadPanda** [comment]
Fair enough - I'll make a submission to add a table of contents to the top of the file to make navigation easier.

**Comment by hyangah** [comment]
Thanks for the future PR.
FYI GitHub UI has this ToC feature, but I agree that it's hard to find. :-(

![Screen Shot 2022-02-17 at 12 29 32 PM](https://user-images.githubusercontent.com/4999471/154538111-95491d55-7713-47a1-aeb2-8a5f464b6b9d.png)
